pkg/client/connector/userd_trafficmgr/traffic_manager.go

```diff
@@ -377,7 +381,10 @@ func (tm *trafficManager) remain(c context.Context) error {
 		case <-ticker.C:
 			_, err := tm.managerClient.Remain(c, &manager.RemainRequest{
 				Session: tm.session(),
-				ApiKey:  func() string { tok, _ := tm.callbacks.GetCloudAPIKey(c, "manager", false); return tok }(),
+				ApiKey: func() string {
+					tok, _ := tm.callbacks.GetCloudAPIKey(c, a8rcloud.KeyDescTrafficManager, false)
```

Comment:
same here. Why is that error ignored?

Reply:
Including an apikey with this request is optional; we might not even be logged in.